docs/contributing.md

```diff
@@ -1,16 +1,33 @@
 # Contributing
-Contributions are welcome. We have a few requests of any new contributors.
+Contributions are welcome. We have a few requests for new contributors:
 
-* Follow styling as designated in our [styling documentation](/docs/style.md).
-* Ensure that all changes which affect logic are covered by unit tests. 
-* Do not introduce any unit test failures/regressions.
-* Turn on automated github actions in your fork to have github run all the unit tests after pushing. See example below:
+* **Follow styling guidelines.**
+  Please take a look at the [code style documentation](/docs/style.md)
+  to ensure uniformity and ease of communication.
```

Comment:
```suggestion
  Please take a look at the [code style documentation](/docs/style.md)
  to ensure ease of communication and uniformity.
```

I don't know why this is standing out to me so much now, but it looks really weird with "ease of communication" second.